### PR TITLE
Make tests easier to run

### PR DESCRIPTION
**Pull request by IvanIvanoff**
## Changes
Remove some tests that use influxdb. Add a flag `influx_store_enabled`
and set it to false in test env so the exporters do not push data to
influx

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have tried to find clearer solution before commenting hard-to-understand parts of code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: test/sanbase_web/graphql/social_data/twitter_api_test.exs [hunk 1]
```diff
@@ -1,196 +0,0 @@
-defmodule Sanbase.Github.TwitterApiTest do
```
Comment (tspenov) [inline test/sanbase_web/graphql/social_data/twitter_api_test.exs:1]: Don't we still need these tests ?

File: lib/sanbase/external_services/coinmarketcap/ticker_fetcher.ex [hunk 2]
```diff
@@ -67,9 +67,11 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.TickerFetcher do
     |> Enum.each(&store_latest_coinmarketcap_data!(&1, cmc_id_to_slugs_mapping))
 
     # Store the data in Influxdb
-    tickers
-    |> Enum.flat_map(&Ticker.convert_for_importing(&1, cmc_id_to_slugs_mapping))
-    |> Store.import()
+    if Application.get_env(:sanbase, :influx_store_enabled, true) do
```
Comment (tspenov) [inline lib/sanbase/external_services/coinmarketcap/ticker_fetcher.ex:70]: I guess that is a temporary thing ?

Comment (IvanIvanoff) [reply lib/sanbase/external_services/coinmarketcap/ticker_fetcher.ex:70]: Until we remove completely influxdb

File: test/sanbase_web/graphql/social_data/twitter_api_test.exs (hunk 1 above)
Comment (IvanIvanoff) [reply test/sanbase_web/graphql/social_data/twitter_api_test.exs:1]: We cannot run these tests without Influx and the goal of this PR is to remove Influxdb as a test dependency.
I'll be moving twitter to CH in the next few days though, so these tests will be deleted either way.

Comment (tspenov) [reply test/sanbase_web/graphql/social_data/twitter_api_test.exs:1]: We can just mock `all_records_for_measurement` and `last_record_for_measurement` and not use influx but keep the tests.
After migrating to CH the api interface will be the same so tests can be reused. (maybe just the function names will change)

Comment (IvanIvanoff) [reply test/sanbase_web/graphql/social_data/twitter_api_test.exs:1]: Added the tests with mocks.

**Pull request merged**